baselines/dom.generated.d.ts

```diff
@@ -14852,6 +14853,59 @@ interface EventSourceInit {
     readonly withCredentials: boolean;
 }
 
+interface AnimationOptions {
+    id?: DOMString;
+    delay?: number;
+    direction?: "normal" | "reverse" | "alternate" | "alternate-reverse";
+    duration?: number;
+    easing?: string;
+    endDelay?: number;
+    fill?: "none" | "forwards" | "backwards" | "both"| "auto";
+    iterationStart?: number;
```

Comment:
missing `iterations: number`

Reply:
ok => commit will be amended accordingly